code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -1603,7 +1603,7 @@
 /datum/reagent/alcohol/amasec/affect_ingest(mob/living/carbon/M, alien, effect_multiplier)
 	. = ..()
 	if(M.stats.getPerk(PERK_GUNSLINGER))
-		M.stats.addTempStat(STAT_TGH, 3, STIM_TIME, "Amasec")
+		M.stats.addTempStat(STAT_TGH, STAT_LEVEL_BASIC*effect_multiplier, STIM_TIME, "Amasec") //Improves Toughness by ~15 if Gunslinger
```

Comment:
```suggestion
		M.stats.addTempStat(STAT_TGH, STAT_LEVEL_BASIC, STIM_TIME, "Amasec") //Improves Toughness by 15 if Gunslinger
```
Effect multiplier does silly math all the time and varies a lot. Let's not have 3.12561561 or 43.1555 stat buffs from this.
Solid and stable 15 is good.

Reply:
```suggestion
		M.stats.addTempStat(STAT_TGH, STAT_LEVEL_BASIC, STIM_TIME, "Amasec")
```
Also, please don't use value of constant in a comment, those may change, and nobody will go trough every comment to update values. Might cause some confusion later down the line.
This code is descriptive enough.

Reply:
Fixed to STAT_LEVEL_BASIC

Reply:
Point about referencing other values in comments still stands.
Values are changing often, thanks to contributors very concerned with game balance.
It makes sense to reference other chemicals with similar effects in a PR, as a justification for changes made, but rarely (actually never) needed in code comments. Capiche?

Reply:
okay, will stop adding in comments.